gensim/models/lsimodel.py

```diff
@@ -489,7 +489,8 @@ def add_documents(self, corpus, chunksize=None, decay=None):
             chunksize = self.chunksize
         if decay is None:
             decay = self.decay
-
+        if corpus == []:
+            warnings.warn('LsiModel.add_documents() called but no documents provided, is this intended?')
```

Comment:
We want standard logging (`logger.warn()`).

Reply:
@mark-todd can you fix this? We're going to be releasing a new version of Gensim soon, and this PR could go in if finished.

Reply:
Ah sure ok - sorry this slipped my mind - when would you need it by?